Cargo.toml

```
@@ -1,76 +1,41 @@
-[package]
-name = "bigerror"
-version = "0.9.0"
+[workspace]
+members = [
+    "bigerror", "bigerror_derive",
+    # "bigerror_dyn",
+]
+resolver = "2"
+
+[workspace.package]
+version = "0.9.0-alpha"
```

Just adding the `alpha` decrements the version. Should the patch version also get updated? 
```suggestion
version = "0.9.1-alpha"
```

---

Sorry, I was thinking about as if it were an RC. Just adding the `alpha` is ok.

---

no worries, we could always increment it by `alpha.1` but I figure we'd do that in `0.9.0-beta.1`